desktop/extensions-sdk/dev/test-debug.md

```diff
@@ -2,16 +2,18 @@
 title: "Step three: Test and debug"
```

Comment:
The title needs to be renamed (I have not checked if there are other places still mentioning step 1 of step 2...)